config/config.go

```diff
@@ -37,6 +37,8 @@ type Config struct {
 		// IsBare if true this repository is assumed to be bare and has no
 		// working directory associated with it
 		IsBare bool
+		// Worktree is the path to the root of the working tree
+		Worktree string
```

Comment:
worktree is a single word, is a concept. I think that is enough to explained that is a path, since a worktree and is a string.